portal/src/main/scala/cromwell/pipeline/ApplicationComponents.scala

```diff
@@ -17,11 +17,12 @@ final class ApplicationComponents(
   val materializer: ActorMaterializer
 ) {
   lazy val applicationConfig: ApplicationConfig = ApplicationConfig.load(config)
+  lazy val utilsModule: UtilsModule = new UtilsModule(applicationConfig)
```

Comment:
Utils definitely should not use configuration files.

Reply:
we use authConfig there. Should I narrow it by AuthCofig(not applcationConfig)?

Reply:
injected AuthConfig only